packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -44,20 +44,24 @@ export interface CrudMoreActionsMenuProps
 }
 
 interface CrudMoreActionsGroupProps {
-    groupTitle: ReactNode;
+    groupTitle?: ReactNode;
     menuListProps?: MenuListProps;
     typographyProps?: ComponentProps<typeof Typography>;
 }
 
 function CrudMoreActionsGroup({ groupTitle, children, menuListProps, typographyProps }: PropsWithChildren<CrudMoreActionsGroupProps>) {
-    return (
-        <>
-            <Typography variant="overline" color={(theme) => theme.palette.grey[500]} sx={{ padding: "20px 15px 0 15px" }} {...typographyProps}>
-                {groupTitle}
-            </Typography>
-            <MenuList {...menuListProps}>{children}</MenuList>
-        </>
-    );
+    if (groupTitle) {
+        return (
+            <>
+                <Typography variant="overline" color={(theme) => theme.palette.grey[500]} sx={{ padding: "20px 15px 0 15px" }} {...typographyProps}>
+                    {groupTitle}
+                </Typography>
+                <MenuList {...menuListProps}>{children}</MenuList>
+            </>
+        );
+    } else {
+        return <MenuList {...menuListProps}>{children}</MenuList>;
+    }
```

Comment:
I'd instead use

```tsx
return (
    <>
        {groupTitle && (
            <Typography variant="overline" color={(theme) => theme.palette.grey[500]} sx={{ padding: "20px 15px 0 15px" }} {...typographyProps}>
                {groupTitle}
            </Typography>
        )}
        <MenuList {...menuListProps}>{children}</MenuList>
    </>
);
```

This removes the duplication of MenuList an prevents bugs when changes are made to only one of the branches.